### PR TITLE
Rename `url` to `custom_url` in Category Model

### PR DESCRIPTION
**Pull request by kzhang-dsg**
According to the APi response, the `url` property should be `custom_url` for Category Model

### CODE REVIEW

**Comment by bc-traciporter** [comment]
@kzhang-dsg Thank you for submitting this PR.  The line you indicated refers to a model that is not called in the specification file. Changing this line would not change what is displayed in our developer docs.

**Comment by bc-traciporter** [comment]
@kzhang-dsg After looking at it again, you are correct.  We will merge your changes.

**Review by bc-traciporter — APPROVED** [review]
LGTM

**Review by slsriehl — CHANGES_REQUESTED** [review]
@kzhang-dsg Until we dedupe the new Catalog files, we have repeated schemas. Please make these changes in the `category-trees_catalog.v3.yml` file as well.

**Comment by bc-traciporter** [comment]
@slsriehl I updated the `categories-trees_catalog.v3.yml file. Can you review it?

File: reference/catalog.v3.yml [hunk 1]
```diff
@@ -21587,7 +21587,7 @@ components:
         - $ref: '#/components/schemas/default_product_sort'
         - type: object
           properties:
-            url:
+            custom_url:
```
Comment (slsriehl) [inline reference/catalog.v3.yml:21590]:
```suggestion
            url:
```

revert

File: reference/catalog.v3.yml [hunk 2]
```diff
@@ -21692,7 +21692,7 @@ components:
     Url:
       type: object
       properties:
-        path:
+        url:
```
Comment (slsriehl) [inline reference/catalog.v3.yml:21695]:
```suggestion
        path:
```
revert for read-only file

**Comment by slsriehl** [comment]
Due to limitations with editing a fork branch and the size of the old catalog file, i'm merging this to a local feature branch so we can revert changes to the read-only file.

**Comment by bc-traciporter** [comment]
Thanks @slsriehl

**Pull request merged**